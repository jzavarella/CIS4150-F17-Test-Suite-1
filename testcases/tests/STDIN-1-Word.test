#!/bin/sh

echo "HELLO" | ./testcases/bin/cmulookup testcases/includes/full-dictionary
