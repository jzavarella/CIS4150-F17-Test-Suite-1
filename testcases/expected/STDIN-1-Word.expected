HH AH0 L OW1
HH EH0 L OW1
DONE! (1 words)
